global.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sdk": {
```

Comment:
I'm generally pretty yucky on `global.json` files. Why do we need it?

If the intention is to get it to work as a nuget tool with .NET 9, I think the better solution is to put `<RollForward>LatestMajor</RollForward>` in the csproj.